### ISSUE TITLE
Can't read the data on the top right corner of scatter graph

### ISSUE THREAD

**Issue by ghostie23** [SpaceGroupUCL/depthmapX#83]
![2017-10-25 2](https://user-images.githubusercontent.com/33085293/31987690-2a1cf482-b99f-11e7-9ba1-b50d70331f04.png)

**Comment by blsemo** [comment]
Just to clarify - the issue is that the letters are cut off at the top so that the data can't be read?

**Comment by ghostie23** [comment]
@blsemo  Yeah, that's the issue
I've tried resetting my screen resolution and custom scaling, none of that works
I even tried to export the graph but I couldn't export it (every time I did, only the image on the left was exported instead when I open it on Ai)

**Comment by varoudis** [comment]
@ghostie23 using newer than 0.50 version? Win10?
I use scatters a lot and Im not getting this (Mac and Win7).

**Comment by ghostie23** [comment]
@varoudis no, I'm using 0.30 version on Win10, I can't seem to install 0.50
